Test.cpp

```diff
@@ -91,7 +91,7 @@ int main(int argc, char const *argv[])
                                           ".",
                                           "./test.log",
                                           "/usr/bin",
-                                          "/usr/lib/llvm/bin"
+                                          "/usr/bin"
                                         );
```

Comment:
These initialisations are traditionally a pain.
Is there a way to get some default values being set via cmake?
(spoiler: yes)
Is there a way to get some default values being set via cmake without writing 3000 lines of cmake?
To be investigated....

Reply:
Yes and yes, I'll push that soon